src/drivers/magnetometer/bmm150/bmm150.cpp

```diff
@@ -676,6 +710,11 @@ extern "C" __EXPORT int bmm150_main(int argc, char *argv[])
 		return ThisDriver::module_custom_method(cli, iterator);
 	}
 
+	if (!strcmp(verb, "selftest")) {
```

Comment:
This is another one (along with reset) that might make sense on the interface. I'd still like to go back and adding the self tests to other new drivers, although I'm not sure if they actually provide much value.

Reply:
I added it because we actually had a failing unit, but it could be because it was an early stage board.